HedgeModManager/Languages/ru-RU.xaml

```diff
@@ -69,11 +69,19 @@
     <system:String x:Key="CodesUIInfo"    >Нажмите два раза по этому коду для большей информации.</system:String>
     <system:String x:Key="CodesUIInfoHint">Наведите курсор на код для большей информации.</system:String>
     <system:String x:Key="CodesUINoInfo"  >Нет информации.</system:String>
+	<system:String x:Key="CodesUICopyCode">Скопировать код в буфер обмена</system:String>
```

Comment:
Fix the white spacing here.

Reply:
![image](https://github.com/thesupersonic16/HedgeModManager/assets/91889396/0db454b5-778b-433b-9d12-a84d8509abff)
What do you mean by "white space"? It looks normal to me in the editor

Reply:
Its using a tabbed space instead of 4 spaces like the rest of them